--- compose/cli/main.py	2021-05-10 12:30:43.000000000 +0200
+++ compose/cli/main.py	2025-01-04 13:18:28.455170376 +0100
@@ -1,10 +1,10 @@
 import contextlib
 import functools
 import json
 import logging
-import pipes
+import shlex
 import re
 import subprocess
 import sys
 from distutils.spawn import find_executable
 from inspect import getdoc
@@ -1577,11 +1577,11 @@
         tls_options.extend(
             ['--context', context]
         )
 
     args = [executable_path] + tls_options + args
-    log.debug(" ".join(map(pipes.quote, args)))
+    log.debug(" ".join(map(shlex.quote, args)))
 
     filtered_env = {k: v for k, v in environment.items() if v is not None}
 
     return subprocess.call(args, env=filtered_env)
 
